### PR TITLE
chore: update footer jan22

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]

File: i18n/locales/english/trending.json [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+  "article7link": "https://www.freecodecamp.org/news/what-is-about-blank-chrome-blank-homepage-guide/",
+  "article8title": "CSS Background Color",
+  "article8link": "https://www.freecodecamp.org/news/css-background-color-how-to-change-the-background-color-in-html/",
+  "article9title": "Basic HTML5 Template",
+  "article9link": "https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/",
+  "article10title": "CSS Border",
+  "article10link": "https://www.freecodecamp.org/news/css-border-style-and-html-code-examples/",
+  "article11title": "SQL Queries",
+  "article11link": "https://www.freecodecamp.org/news/learn-sql-queries-database-query-tutorial-for-beginners/",
+  "article12title": "HTML <a> tag",
```
Comment (ojeytonwilliams) [inline i18n/locales/english/trending.json:26]: Here's the issue.  The `<a>` is getting parsed as HTML and creating a new element `<a> tag </a>`.  @scissorsneedfoodtoo can we use something like https://www.npmjs.com/package/html-entities to encode these or do we have to do it by hand?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: i18n/locales/english/trending.json (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply i18n/locales/english/trending.json:26]: Hey @ojeytonwilliams, thanks for flagging me and for pointing that package out.

I'm thinking that we can escape these selectively within the `footer.njk` template itself using the `escape` filter that's built into Nunjucks. 

I originally thought that any HTML tags included in each locale's `*.json` files would automatically be escaped by i18next, but that doesn't seem to be the case here.

I think the easiest thing we can do for these footer titles is to set a variable for each one, then use the built-in Nunjucks `escape` filter to render each title variable to the template:

```
{%- set article12title %}{% t 'trending:article12title' %}{% endset -%}
{{ article12title | escape }}
```

Also, the `{%-` and `-%}` should get rid of the extra whitespace, which I missed when first porting the footer over.

Comment (ojeytonwilliams) [reply i18n/locales/english/trending.json:26]: Great! Nunjuck's built in escape sounds like the way to go.

**Review by ojeytonwilliams — APPROVED** [review]
image

Nice one, Kris. LGTM :+1:

**Review by raisedadead — APPROVED** [review]

**Pull request merged**